workers/notifier/src/main/kotlin/notifier/NotifierRunner.kt

```diff
@@ -20,7 +20,8 @@
 package org.eclipse.apoapsis.ortserver.workers.notifier
```

Comment:
Could you update the commit message to reflect that this change is not only about using the mail and jira configuration the `AdminConfig`, but also about using the `notifierRules` and `rulesSet` from the `AdminConfig` to read the corresponding files? It's not clear from the commit message.